src/lib/os-release.coffee

```diff
@@ -0,0 +1,32 @@
+Promise = require 'bluebird'
+fs = require 'fs'
+_ = require 'lodash'
+
+exports.getOSReleaseField = (path, field) ->
+	try
+		releaseData = fs.readFileSync(path)
+		lines = releaseData.toString().split('\n')
+		releaseItems = {}
+		for line in lines
+			[ key, val ] = line.split('=')
+			releaseItems[_.trim(key)] = _.trim(val)
+		# Remove enclosing quotes: http://stackoverflow.com/a/19156197/2549019
+		return releaseItems[field].replace(/^"(.+(?="$))"$/, '$1')
```

Comment:
This on ealready does that (check the stackverflow link), and has been tested in production, so I'd say let's stick with it